Formula/mediaconch.rb

```diff
@@ -36,7 +36,7 @@ def install
               "--disable-dependency-tracking",
               "--enable-static",
               "--enable-shared",
-              "--with-libcurl",
+              "--with-libcurl=runtime",
```

Comment:
@g-maxime Homebrew provides a libcurl.pc file https://github.com/Homebrew/brew/blob/master/Library/Homebrew/os/mac/pkgconfig/10.12/libcurl.pc so it should be able to use that.